lib/ambientlight/simulator.js

```diff
@@ -24,7 +24,7 @@ function Simulator () {
   };
 }
 
-Simulator.prototype.read = function (callback) {
+Simulator.prototype.readLight = function (callback) {
```

Comment:
ok you aligned to sensor good maybe sensor should use read, but upstream is not very reactive

please squash this change in similator.js change

Reply:
I intend to rebase everything into a single commit once it's ready for merge to have a clear commit easy to revert.